track/track.py

```diff
@@ -319,6 +332,35 @@ def __repr__(self):
     def __len__(self):
         return len(self.bounds_history)
 
+    def get_best_human_track(track_meta, tag_precedence, min_confidence=-1):
+        """ returns highest precidence non AI tag from the metadata """
+
+        track_tags = track_meta.get("TrackTags")
+        if not track_tags:
+            return None
+        track_tags = [
+            tag
+            for tag in track_tags
+            if not tag.get("automatic", False)
+            and tag.get("confidence") > min_confidence
+        ]
+        tag = None
+        if track_tags:
+            default_prec = tag_precedence.get("default", 100)
+            tag = min(
+                track_tags,
+                key=lambda tag: Track.tag_order(tag, tag_precedence, default_prec),
+            )
```

Comment:
If there's multiple equal confidence human tags (i.e. people couldn't agree) perhaps None should also be returned?
In this case it's safer to not train with the track as it could mislead the classifier.

Reply:
ye sounds good, have updated to use a loop instead.